cmd/istioctl/main.go

```diff
@@ -316,3 +323,36 @@ func readInputs() ([]inputDoc, error) {
 
 	return varr, nil
 }
+
+// Print a simple list of names
+func printShortOutput(list map[model.Key]proto.Message) error {
+	for key := range list {
+		fmt.Printf("%v\n", key.Name)
+	}
+
+	return nil
+}
+
+// Print as YAML
+func printYamlOutput(list map[model.Key]proto.Message) error {
+	for key, item := range list {
+		out, err := schema.ToYAML(item)
+		if err != nil {
```

Comment:
Probably better to return the `err` if `schema.ToYAML` failed so error can be propagated to os.Exit. Otherwise, the program would exit successfully printing non-valid output.